physical/postgresql_test.go

```diff
@@ -0,0 +1,42 @@
+package physical
+
+import (
+	"os"
+	"testing"
+
+	_ "github.com/lib/pq"
+)
+
+func TestPostgreSQLBackend(t *testing.T) {
+	connURL := os.Getenv("PGURL")
```

Comment:
Sorry, just realised these environment variables are only used in the tests! (It's been a long day!)